code/game/atoms_movable.dm

```diff
@@ -1040,6 +1042,25 @@
 		return
 	DISABLE_BITFIELD(flags_pass, HOVERING)
 
+///returns bool for if we want to get forcepushed
+/atom/movable/proc/force_pushed(atom/movable/pusher, force = MOVE_FORCE_DEFAULT, direction)
```

Comment:
Why is this a proc

Reply:
TG port its so damage and etc can be handled as needed